src/components/Slider/Slider.tsx

```diff
@@ -38,12 +50,18 @@ const Slider: FC<PropsWithChildren<SliderProps>> = (props) => {
     navigationButtonsShowType = 'hide',
     spaceBetween = 0,
     showPaginationText,
+    onSlideIndexChange,
+    initialSlide,
   } = currentProps;
+
+  const [slideIndex, setSlideIndex] = useState(initialSlide || 0);
```

Comment:
handle navigate to slide when initial slide passed

Reply:
✅